### ISSUE TITLE
Failed to mount folders in Linux guest

### ISSUE THREAD

**Issue by ocafebabe** [dotless-de/vagrant-vbguest#221]
It seems that openSUSE boxes are broken with the latest vbguest release (0.12.0)!

I get this error when building a new virtual machine: **vagrant_vbguest.machine_loop_guard**

And then I see this in the console:

```bash
Failed to mount folders in Linux guest. This is usually because
the "vboxsf" file system is not available. Please verify that
the guest additions are properly installed in the guest and
can work properly. The command attempted was:
mount -t vboxsf -o uid=`id -u vagrant`,gid=`getent group vagrant | cut -d: -f3` vagrant /vagrant
mount -t vboxsf -o uid=`id -u vagrant`,gid=`id -g vagrant` vagrant /vagrant
The error output from the last command was:
/sbin/mount.vboxsf: mounting failed with the error: No such device```

**Comment by inardin** [comment]
The same for CentOS 6 guests.

**Comment by glookie1** [comment]
This is the exact same error that appears if you don't have any vboxguest additions installed.

Are there any error messages before this one?

**Comment by ocafebabe** [comment]
No other errors before this one!

But I just noticed that a new version (1.0.1) of the openSUSE box that I use is listed [here](https://atlas.hashicorp.com/opensuse/boxes/openSUSE-13.2-x86_64) with this note: "**Fixed loading of vboxfs module**"!

It seems that the error with the latest vbguest plugin has been fixed but this version hasn't been released yet...

**Comment by bobweston** [comment]
Getting the exact error on Ubuntu 14.04. If I manually log in to the guest and run this command, I can then provision the box. 

  sudo ln -s /opt/VBoxGuestAdditions-4.3.22/lib/VBoxGuestAdditions /usr/lib/VBoxGuestAdditions

But if I destroy the box and try to rebuild, I'm back to square one.

There is one difference in my case, I see an error: 

<pre>
An error occurred during installation of VirtualBox Guest Additions 5.0.14. Some functionality may not work as intended.
</pre>

<pre>
vagrant_vbguest.machine_loop_guard

Installing Virtualbox Guest Additions 5.0.14 - guest version is 4.3.22
</pre>

Then I get the above mentioned Failed to mount folders in Linux Guest.

Anyone else seeing this?

**Comment by ocafebabe** [comment]
Issue seems to be resolved with the latest plugin release (0.13.0)!
